### PR TITLE
Makes change to print out votes from 1-1024 instead of 0-1023

### PR DESCRIPTION
**Pull request by srinitude**
@papamuziko Made a small change to my code to make a PR

### CODE REVIEW

File: level_2/1024.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def cast_vote(url, cookie, payload):
```
Comment (papamuziko) [inline level_2/1024.py:44]: `endpoint` is a constant, it should be in uppercase: `ENDPOINT`

File: level_2/1024.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def cast_vote(url, cookie, payload):
 
 get_initial_data(endpoint, hodor_cookie, hodor_payload)
```
Comment (papamuziko) [inline level_2/1024.py:45]: Use `if __name__ == "__main__":` to make your code executable only when call it directly and not when import the file

Comment (papamuziko) [inline level_2/1024.py:45]: Better to create a Class and store attributes instead of storing global variables

Comment (srinitude) [reply level_2/1024.py:45]: @papamuziko This is actually what I wanted to ask. What kind of class should I create to encapsulate all of this data and behavior?

**Pull request closed without merging**